src/serac/infrastructure/output.cpp

```diff
@@ -50,12 +49,10 @@ void outputFields(const axom::sidre::DataStore& datastore, const std::string& fi
   extracts["e1/params/protocol"] = output_language;
 
   // Get domain Sidre group
-  // TODO: get this from StateManager directly?
-  const axom::sidre::Group* sidre_root      = datastore.getRoot();
-  const std::string         collection_name = StateManager::collectionName();
-  SLIC_ERROR_ROOT_IF(!sidre_root->hasGroup(collection_name),
-                     fmt::format("Expected a datacollection root at '{0}' but it was not found", collection_name));
-  const axom::sidre::Group* domain_grp = sidre_root->getGroup(collection_name);
+  const axom::sidre::Group* sidre_root = datastore.getRoot();
+  SLIC_ERROR_ROOT_IF(!sidre_root->hasGroup(data_collection_name),
+                     fmt::format("Expected a datacollection root at '{0}' but it was not found", data_collection_name));
```

Comment:
Maybe adding `in sidre` or something to know what we are looking at?

Reply:
No, the only functional change was the breaking the circular dependancy.  This happened in output.cpp/hpp and in serac.cpp source files